meshroom/ui/qml/ImageGallery/ImageGallery.qml

```diff
@@ -292,6 +292,24 @@ Panel {
 
                     function sendRemoveRequest() {
                         if (!readOnly) {
+                            // look if the viewpoint's intrinsic is used by another viewpoint
```

Comment:
Should be done after the removeImageRequest, isn't it?

Reply:
Maybe better to do it on the python side.
Instead of a generic call to reconstruction.removeAttribute(attribute), here:
https://github.com/alicevision/Meshroom/blob/7b691a05dd73c1aa1ee913aa11af84ba81d227b5/meshroom/ui/qml/WorkspaceView.qml#L81C17-L81C80

Add a removeImage and do all the checks in it (including the job done in allViewpointsCleared / removeAllImages).
https://github.com/alicevision/Meshroom/blob/7b691a05dd73c1aa1ee913aa11af84ba81d227b5/meshroom/ui/graph.py#L820C9-L820C24